OpenRA.Mods.Common/Traits/CaptureManager.cs

```diff
@@ -40,6 +40,20 @@ public class CaptureManagerInfo : ITraitInfo
 		public readonly bool PreventsAutoTarget = true;
 
 		public virtual object Create(ActorInitializer init) { return new CaptureManager(this); }
+
+		public bool CanBeTargetedBy(FrozenActor frozenActor, Actor captor, Captures captures)
+		{
+			if (captures.IsTraitDisabled)
+				return false;
+
+			// TODO: FrozenActors don't yet have a way of caching conditions, so we can't filter disabled traits
+			// This therefore assumes that all Capturable traits are enabled, which is probably wrong.
+			// Actors with FrozenUnderFog should therefore not disable the Capturable trait.
```

Comment:
Wouldn't it be good to mention this in the `Desc` (of `Capturable`)?

Reply:
This has always been the case, so i'm not too inclined to change this now.

Reply:
Done.